### PR TITLE
man page SAPHanaSR.py.7: example test

### PR DESCRIPTION
**Pull request by lpinne**
Hi,
I have aligned man page SAPHanaSR.py.7 with angi´s susHanaSR.py.7.
Not urgent, just would be nice to have it in one of the next updates.
Regards,
Lars

### CODE REVIEW

File: man/SAPHanaSR.py.7 [hunk 1]
```diff
@@ -269,6 +309,7 @@ Please report any other feedback and suggestions to feedback@suse.com.
 \fBSAPHanaSR-monitor\fP(8) , \fBSAPHanaSR-showAttr\fP(8) ,
 \fBSAPHanaSR-manageProvider\fP(8) , \fBSAPHanaSR-hookHelper\fP(8) ,
 \fBcrm_attribute\fP(8) , \fBsudo\fP(8) , \fBsudoers\fP(5), \fBpython\fP(8) ,
+\fBitables\fP(8) , \fBss\fP(8) ,
```
Comment (angelabriel) [inline man/SAPHanaSR.py.7:312]: Typo? iptables instead of itables?

**Review by angelabriel — CHANGES_REQUESTED** [review]
Can you please check, if there is a typo in line 312

File: man/SAPHanaSR.py.7 (hunk 1 above)
Comment (lpinne) [reply man/SAPHanaSR.py.7]: thanks!
fixed - also in my local angi branch

**Review by angelabriel — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**